dali/pipeline/operators/crop/bbox_crop.h

```diff
@@ -156,8 +154,10 @@ class BBoxCrop : public Operator<CPUBackend> {
       auto *output = bbox_out_data + i * kBboxSize;
       output[0] = bounding_boxes[i].left;
       output[1] = bounding_boxes[i].top;
-      output[2] = bounding_boxes[i].right;
-      output[3] = bounding_boxes[i].bottom;
+      output[2] = ltrb_ ? bounding_boxes[i].right
+                        : bounding_boxes[i].right - bounding_boxes[i].left;
+      output[3] = ltrb_ ? bounding_boxes[i].bottom
+                        : bounding_boxes[i].bottom - bounding_boxes[i].top;
```

Comment:
Probably a good idea for later work would be to hide this with getters in Rectangle, so regardless of underlying storage you could get left() or width()